### PR TITLE
Org list page

### PR DESCRIPTION
**Pull request by cameronmarsh**

### CODE REVIEW

File: apps/andi/lib/andi/input_schemas/organizations.ex [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
+
+  def get_all(), do: Repo.all(Organization)
+
+  def get_harvested_dataset(id) do
     Repo.get(HarvestedDatasets, id)
     |> HarvestedDatasets.preload()
   end
 
-  def get_all() do
+  def get_all_harvested_datasets() do
```
Comment (bbrewer-forge) [inline apps/andi/lib/andi/input_schemas/organizations.ex:22]: I'd like to see this be there with an `org_id` argument, as that seems like the most common case we'll have, but feel free to burn that bridge when we get to it.

File: apps/andi/lib/andi_web/live/organization_live_view.ex [hunk 2]
```diff
@@ -0,0 +1,126 @@
+defmodule AndiWeb.OrganizationLiveView do
```
Comment (bbrewer-forge) [inline apps/andi/lib/andi_web/live/organization_live_view.ex:1]: I'd love to see us take a crack at making this not repeat some of the problems with the datasets page (Ex. use the Repo + Query to do searching as it's arguably faster or at least doesn't kill the client machine with javascript based filtering). However, I'm fine with pushing forward if that's too much to pack on there.

File: apps/andi/lib/andi/input_schemas/organizations.ex (hunk 1 above)
Comment (cameronmarsh) [reply apps/andi/lib/andi/input_schemas/organizations.ex:22]: yup makes sense

File: apps/andi/lib/andi/event_handler.ex [hunk 3]
```diff
@@ -37,6 +37,10 @@ defmodule Andi.EventHandler do
     organization_update()
     |> add_event_count(data.id)
 
+    data
```
[the remark this answers is not in the document]
Comment (cameronmarsh) [reply apps/andi/lib/andi/event_handler.ex:40]: no

File: apps/andi/lib/andi_web/live/organization_live_view.ex (hunk 2 above)
Comment (cameronmarsh) [reply apps/andi/lib/andi_web/live/organization_live_view.ex:1]: yeah good call

**Comment by bbrewer-forge** [comment]
WoooOOOooow

**Pull request merged**